### PR TITLE
feat(prefer-const): changes acc. to prefer-const eslint rule

### PR DESCRIPTION
**Pull request by divyabiyani**
Changes w.r.t https://github.com/mozilla/eslint-plugin-fxa/pull/5.
Fixes https://github.com/mozilla/fxa-content-server/issues/4632

### CODE REVIEW

File: app/scripts/lib/experiment.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ define(function (require, exports, module) {
         return;
       }
 
-      for (let experimentName in this._allExperiments) {
+      for (const experimentName in this._allExperiments) {
```
Comment (shane-tomlinson) [inline app/scripts/lib/experiment.js]: Will this work? I was under the impression `let` had to be used inside a for expression.

**Comment by vladikoff** [comment]
@divyabiyani hm so the issue opened asks to check the server code (`server/` directory), did ESLint skip the server code with this fix applied?

File: app/scripts/lib/experiment.js (hunk 1 above)
Comment (vladikoff) [inline app/scripts/lib/experiment.js]: Is this what you mean: https://codepen.io/vladikoff/pen/MJqLjK?editors=1111 

Seems fine, see console output

Comment (vladikoff) [inline app/scripts/lib/experiment.js]: Better link: https://codepen.io/vladikoff/pen/LxJqxe?editors=1111

**Comment by divyabiyani** [comment]
@vladikoff: It did not skip server code.
Server code uses var to initialize it. This rule just works on let.

File: app/scripts/lib/experiment.js (hunk 1 above)
Comment (TDA) [inline app/scripts/lib/experiment.js]: @shane-tomlinson, CMIIW, but I think that is preferable for the traditional for loop, like so:
`for(let i = 0; i < 5; i++)`
Using const should work fine with the for-in loop and the for-of loop.

**Comment by vladikoff** [comment]
@divyabiyani This looks good! This should be ready to be merged, but we need another PR that adds the `no-var`rule, but only applies it to the `server/` directory. Which should result in around ~140 changes to `var` statements....

**Comment by vladikoff** [comment]
@divyabiyani I think the change to `no-var` should just be local to the fxa-content-server repo, so you don't need to modify the eslint-plugin-fxa

**Comment by vladikoff** [comment]
@divyabiyani thank you!!

**Pull request merged**